easybuild/easyblocks/s/snphylo.py

```diff
@@ -76,11 +77,16 @@ def install_step(self):
         """Install by copying files/directories."""
         bindir = os.path.join(self.installdir, 'bin')
         binfiles = ['snphylo.sh', 'snphylo.cfg', 'snphylo.template']
+
+        predir = ''
+        if LooseVersion(self.version) >= LooseVersion('20160204'):
+            predir = 'SNPhylo'
+
         try:
             mkdir(bindir, parents=True)
             for binfile in binfiles:
-                shutil.copy2(os.path.join(self.builddir, binfile), bindir)
-            shutil.copytree(os.path.join(self.builddir, 'scripts'), os.path.join(self.installdir, 'scripts'))
+                shutil.copy2(os.path.join(self.builddir, predir, binfile), bindir)
+            shutil.copytree(os.path.join(self.builddir, predir, 'scripts'), os.path.join(self.installdir, 'scripts'))
```

Comment:
@vanzod Let's also use `copy_file` and `copy_dir` while we're touching this easyblock (and remove the `try`/`except`):

```python
mkdir(bindir, parents=True)
for binfile in binfiles:
    copy_file(os.path.join(self.builddir, binfile), bindir)
copy_dir(os.path.join(self.builddir, predir, 'scripts'), os.path.join(self.installdir, 'scripts'))